src/Web.cpp

```diff
@@ -1685,34 +1685,81 @@ bool DumpNvsToJSONCallback(const char *key, void *data) {
 	return tagIdToJSON(key, obj);
 }
 
+// callback for writing a NVS entry to list
+bool DumpNvsToArrayCallback(const char *key, void *data) {
+	std::vector<String> *keys = (std::vector<String> *) data;
+	keys->push_back(key);
+	return true;
+}
+
+static String tagIdToJsonStr(const char *key) {
+	StaticJsonDocument<512> doc;
+	JsonObject entry = doc.createNestedObject(key);
+	if (!tagIdToJSON(key, entry)) {
+		return "";
+	}
+	String serializedJsonString;
+	serializeJson(entry, serializedJsonString);
+	return serializedJsonString;
+}
+
+uint16_t nvsIndex;
```

Comment:
Move `nvsIndex` either as a static variable into the lambda in line 1736 or at least make the variable `static` (so that it does not pollute the global namespace).